### ISSUE TITLE
Container images used (for boefjes) should use image digests

### ISSUE THREAD

**Issue by SafPlusPlus** [minvws/nl-kat-coordination#3261]
**Describe the bug**

A number of boefjes use 3rd party container images with the "latest" tag, for instance:
- kat_ssl_scan -> breezethink/sslscan:latest
- kat_nmap_ports -> instrumentisto/nmap:latest
- kat_webpage_capture -> mcr.microsoft.com/playwright:latest

If those container registries were to be taken over by an adversary then untrusted 3rd party code would be running potentially as root in containers on the server(s) running the boefjes.

**To Reproduce**

1. Do a basic install
2. Run one of those mentioned boefjes
3. Conclude that it pulled a "latest" tag of an image

**Expected behavior**

I expect this risk to be mitigated by the boefjes in this repository to pull the images, not based on just a tag, but (also) on an image digest. For instance, instead of using `instrumentisto/nmap:latest` using `instrumentisto/nmap:7@sha256:59e2c0bb2e001b14e453cebdf0ad1ba2ac907f4a14492ed67393e8c5bda96539`, which both uses a version tag and a hash digest.

**Screenshots**

n/a

**OpenKAT version**

v1.15.1

**Desktop (please complete the following information if relevant):**

n/a

**Additional context**

n/a

**Comment by underdarknl** [comment]
Agreed. All boefjes should *if possible* use a version pinned image.
